src/backend/sev/runtime.rs

```diff
@@ -1,13 +0,0 @@
-// SPDX-License-Identifier: Apache-2.0
-
-use std::io::Result;
-use std::os::unix::io::AsRawFd;
-
-use super::ioctl::{KvmEncRegion, KvmUserspaceMemoryRegion, ENCRYPT_REGION};
-
-pub fn mark_encrypted(kvm_fd: &impl AsRawFd, region: &KvmUserspaceMemoryRegion) -> Result<()> {
-    let enc_region = KvmEncRegion::new(region);
-    ENCRYPT_REGION
-        .ioctl(&mut kvm_fd.as_raw_fd(), &enc_region)
-        .map(|_| ())
-}
```

Comment:
Oops... here it hides... tststs..
@connorkuehl any idea, why this was never part of the sev / sev-iocuddle / snp crate?

Reply:
In the meantime, I reinvented the wheel with https://github.com/enarx/sev-iocuddle/pull/7

Reply:
It was my hope to keep sev / sev-iocuddle / snp agnostic of any KVM crate. In hindsight, I should have done what you did and passed in a `&[u8]` instead of a `&KvmUserspaceMemoryRegion` and then it would have remained agnostic but the launcher API could have exposed this nicely.